tests/utils/mocked_pagers.rs

```diff
@@ -17,10 +17,20 @@ fn get_mocked_pagers_dir() -> PathBuf {
 /// On Unix: 'most' -> 'most'
 /// On Windows: 'most' -> 'most.bat'
 pub fn from(base: &str) -> String {
-    if cfg!(windows) {
-        format!("{}.bat", base)
-    } else {
-        String::from(base)
+    let mut cmd_and_args = base.split(" ");
```

Comment:
Why do we need to check for `" "` here? Isn't `base` only ever "more", "less", or "echo"?

Reply:
Nevermind, pretend you never saw this comment (I will delete it now)